app/models/edition.rb

```diff
@@ -699,6 +699,10 @@ def has_legacy_tags?
 
   delegate :locked?, to: :document
 
+  def attribute_names
+    @attributes.keys
+  end
```

Comment:
Can we test if we can remove https://github.com/alphagov/whitehall/pull/6483/commits/c20279853ddef4ff69581b27880cd1febef2360a thanks to this?

Reply:
Ah, yes we can. I'll test 49b2aa3 and ad1c9bf too.

Reply:
Seems like issue with `Role`/`RoleAppointment` is unrelated, still occurs even with this change in place (they don't inherit from `Edition`, but moving the definition to `ApplicationRecord` or adding it to `Role` etc doesn't help either).